app/controllers/org_admin/templates_controller.rb

```diff
@@ -38,6 +38,7 @@ def index
     # -----------------------------------------------------
     def new
       authorize Template
+      @current_tab = params[:r] || 'all-templates'
```

Comment:
@current_tab could be refactored into a private method since it seems to be repeated across different actions.
```
def current_tab=(value=params[:r])
  @current_tab =value || 'all-templates'
end
```